db/migrate/20171114225311_create_tournaments.rb

```diff
@@ -0,0 +1,10 @@
+class CreateTournaments < ActiveRecord::Migration[5.1]
+  def change
+    create_table :tournaments do |t|
+      t.integer :game_mode
```

Comment:
I have actually never needed to use enums yet, but they seem a good idea. Based on [this article](https://hackhands.com/ruby-on-enums-queries-and-rails-4-1/) using a column like: `t.integer :game_mode, default: 0, null: false` seems like a good idea to avoid null checks further down the line.